command/cp.go

```diff
@@ -177,6 +177,12 @@ func NewSharedFlags() []cli.Flag {
 			Name:  "content-encoding",
 			Usage: "set content encoding for target: defines content encoding header for object, e.g. --content-encoding gzip",
 		},
+		&cli.IntFlag{
+			Name:        "no-such-upload-retry-count",
+			Usage:       "number of times that a request will be retried on NoSuchUpload error. You should not use this unless you really know what you're doing.",
```

Comment:
Looks like we use all lower case in usage texts -except abbrevations-, let's not use upper case to start a new sentence to make it consistent.  `;` might be used to separate sentences as done in `sse-kms-key-id` flag.